### PR TITLE
Jakarta SOAP with Attachments 1.4

### PR DESCRIPTION
**Pull request by lukasj**
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat_x.y.pdf
- [x] Specification HTML in the form of wombat_x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://oss.sonatype.org/content/repositories/staging/jakarta/xml/soap/jakarta.xml.soap-api/1.4.2/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-soap-tck-1.4.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/saaj-api/issues/49, https://github.com/eclipse-ee4j/saaj-api/issues/51

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

**Comment by lukasj** [comment]
this one should be ready too; the only thing is that the compatible implementation has no name, so I used "Jakarta SOAP with Attachments Implementation"

**Comment by kwsutter** [comment]
Just putting a "hold" tag on this issue since it hasn't been decided whether the jax-ws specs will be part of Jakarta EE 9 or not...

**Comment by bshannon** [comment]
> Just putting a "hold" tag on this issue since it hasn't been decided whether the jax-ws specs will be part of Jakarta EE 9 or not...

We can approve this spec whether it's included in Jakarta EE 9 or not, right?
Approving this (and other related) spec allows it to be used in standalone
implementations and Jakarta EE implementations independently of the JCP.

**Comment by lukasj** [comment]
TODOs: upload refreshed binaries, fix TCK URL

**Comment by lukasj** [comment]
binaries were refreshed

File: soap-attachments/1.4/_index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Jakarta SOAP with Attachments Specification 1.4"
+date: 2019-07-23T14:30:32+0200
+summary: "Initial release of Jakarta SOAP with Attachments"
+---
+Jakarta SOAP with Attachments defines an API enabling developers to produce
+and consume messages conforming to the SOAP 1.1, SOAP 1.2, and SOAP Attachments Feature.
+
+* [Jakarta SOAP with Attachments 1.4 Specification Document](./jakarta.xml.soap-spec-1.4.pdf) (PDF)
```
Comment (kwsutter) [inline soap-attachments/1.4/_index.md]: Similar comment as the other PRs...  The name of the spec should be soap-spec-1.4.pdf to be consistent with our [naming doc.](https://github.com/jakartaee/specification-committee/blob/master/names.adoc)

Comment (lukasj) [reply soap-attachments/1.4/_index.md]: done

**Comment by kwsutter** [comment]
- [ ] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification and apidocs PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request.
 - [ ] The compatible implementation project/vendor MUST send an email tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as appropriate for the branch management for the project.

**Comment by lukasj** [comment]
@kwsutter unchecked items were done

**Comment by kwsutter** [comment]
Thanks!

**Pull request merged**